### PR TITLE
Add Pupper logo

### PR DESCRIPTION
**Pull request by pupperdao**
#### Hi! What is your PR about?

- [x] Add images for your space
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [x] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
- I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/pupper.eth
- The images file size should not exceed 50KB. Please reduce their weight.
- Your space name directory should be similar to your ENS. Follow this example `spaces/pupper.eth/logo.png`.

**Comment by pupperdao** [comment]
Whoops! Just changed everything to reflect Pupperdao.eth and PupperDao name. Thanks!

File: skins/pupperdao.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.pupper {
```
Comment (gokaiorg) [inline skins/pupperdao.scss:1]: The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.pupperdao {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by pupperdao** [comment]
Should be fixed!

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**